src/ontology/reports/omim_exclusion_reasons.robot.template.tsv

```diff
@@ -1,9 +1,11 @@
 term_id	exclusion_reason
```

Comment:
Ugh, so this is more work for me but I feel like I should prioritize other stuff before this. However, that's up to @matentzn 

Here's the problem I noticed. The following hypothesis proved to be false.

Hypothesis: If I add `SO:<gene>` to `config/omim_exclusions.tsv`, and and run the exclusion term expansion, I get _outputs A_. Then if I remove all of the other OMIM terms in that file which had the exclusion reason `MONDO:excludeGene` and rerun, I get _outputs B_. Outputs _A_ and _B_ should be exactly the same.

But they were not the same. I could do an analysis of the differences between the lists and look further, but I will wait on that.
[before and after.zip](https://github.com/monarch-initiative/mondo-ingest/files/10290792/before.and.after.zip)

Reply:
Never mind. I don't know what happened in this first run, but I re-ran, and I see that there is a small difference, but it is an expected one. there was a gene term that was previously missing from config/omim_exclusions.tsv, and it makes sense that it is showing in the output since it is a descendant of SO:<gene>